src/cocoa/toga_cocoa/widgets/optioncontainer.py

```diff
@@ -48,5 +53,31 @@ def add_content(self, label, widget):
         item.view = widget.native
         self.native.addTabViewItem(item)
 
+    def remove_content(self, index):
+        try:
+            tabview = self.native.tabViewItemAtIndex(index)
+            self.native.removeTabViewItem(tabview)
+        except Exception as e:
+            print(e)
+
     def set_on_select(self, handler):
         pass
+
+    def set_option_enabled(self, index, enabled):
+        tabview = self.native.tabViewItemAtIndex(index)
+        if not enabled and tabview == self.native.selectedTabViewItem:
+            # Don't allow to disable selected tab
+            raise Exception('Disable selected Option is not allowed')
```

Comment:
A generic `Exception` is difficult to handle down the line; We should introduce a `OptionException` (as a subclass of `ValueError`); the error message can then give the reasons that the tab can't be enabled/disabled/deleted.